### PR TITLE
V1 migration Issues 1 to 4

### PR DESCRIPTION
**Pull request by phlexib**
> 1. update request path to use v1 in` API.ts`
> 2. we didn't have a views API before so users can just use the table API 😂
>     - note the new tableType property in `models/Table.ts` comment
>     -  add something to README examples that shows tableType can be access to tell if it's a table or view
> 3. remove all references to Folders API
> 4. convert all references of Sections to Pages (eg. getPage(...) instead of getSection(...))

### CODE REVIEW

**Comment by parker-codes** [comment]
Hey @phlexib, thanks for contributing! I'll review this soon!

File: src/index.ts [hunk 1]
```diff
@@ -36,30 +36,19 @@ export class Coda {
...
-    // https://coda.io/developers/apis/v1beta1#operation/listFolders
-    const { data } = await this.API.request(`/docs/${docId}/folders`, params);
-    return data.items.map((folder) => new Folder({ ...folder, docId })); // map all items into folders
-  }
-
-  async getFolder(docId: string, folderIdOrName: string): Promise<Folder> {
-    // https://coda.io/developers/apis/v1beta1#operation/getFolder
-    const { data } = await this.API.request(`/docs/${docId}/folders/${folderIdOrName}`);
-    return new Folder({ ...data, docId });
+  async getPage(docId: string, PageIdOrName: string): Promise<Page> {
```
Comment (parker-codes) [inline src/index.ts]: Can you make this variable camelcase (`pageIdOrName`)?

File: src/index.ts [hunk 2]
```diff
@@ -36,30 +36,19 @@ export class Coda {
     return new Doc(this.API, data);
   }
 
-  async listSections(docId: string, params: any): Promise<Section[]> {
+  async listPages(docId: string, params: any): Promise<Page[]> {
     // params: limit, pageToken
-    // https://coda.io/developers/apis/v1beta1#operation/listSections
-    const { data } = await this.API.request(`/docs/${docId}/sections`, params);
-    return data.items.map((section) => new Section({ ...section, docId })); // map all items into sections
+    // https://coda.io/developers/apis/v1beta1#operation/listPages
```
Comment (parker-codes) [inline src/index.ts]: Could you update these links to point to `v1` instead of `v1beta1`? Thanks!

File: src/models/index.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-import Row from './Row';
-import Column from './Column';
-import Section from './Section';
-import Folder from './Folder';
-import Control from './Control';
+import Doc from "./Doc";
+import Table from "./Table";
+import Row from "./Row";
+import Column from "./Column";
+import Page from "./Page";
```
Comment (parker-codes) [inline src/models/index.ts]: Can you remove this empty line between the model imports?

**Review by parker-codes — COMMENTED** [review]
There are just a few small things I'm hoping you could update. This all looks good! 

Normally it's good to ask permission before making so many code-style changes (like splitting method params onto multiple lines), but I'm okay with them. They may get reverted in the future though.

**Comment by phlexib** [comment]
Hi, 
thanks for the comments , it's my first PR so I totally welcome all of your comments.
The code-style changes are my ESLint settings on auto-save. I noticed it changed all the single quote to double quote and fixed it. But I didn't think the multilines would be a problem. Sorry

**Comment by phlexib** [comment]
Here's a new commit with all your note : [v1-migration](https://github.com/parker-codes/coda-js/pull/15/commits/6e017fc611fe7d1266a1eab1a77076802b7ef363)

**Comment by parker-codes** [comment]
@phlexib It looks great. Thanks again! 😁

**Comment by parker-codes** [comment]
@phlexib I actually reverted the merge. The code still looks good, but I realized that you had the PR pointing to the `master` branch. Could you create a new PR from your `v1-migration` branch to this repos `v1-migration` branch?

**Pull request merged**